service/src/main/java/bio/terra/workspace/service/resource/referenced/flight/clone/SetCloneReferencedResourceResponseStep.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.workspace.service.resource.referenced.flight.clone;
+
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.common.utils.FlightUtils;
+import bio.terra.workspace.service.resource.referenced.model.ReferencedResource;
+import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys;
+import org.springframework.http.HttpStatus;
+
+public class SetCloneReferencedResourceResponseStep implements Step {
```

Comment:
I didn't see where this step was used. Maybe I missed it. Is it used?

Reply:
It was, but it's not used anymore. Removed.